doc/installation.rst

```diff
@@ -8,36 +8,95 @@
 Installation
 ##########################################
 
-The subsequently described installation steps are demonstrated as shell commands, where the path before the ``%`` sign denotes the directory in which the commands following the ``%`` should be entered.
+Before installing PyPSA-Earth on your computer, it's crucial to ensure that your system meets the necessary hardware and software requirements. The following sections outline the prerequisites in terms of hardware and software. Additionally, detailed installation guidelines for required software tools will be provided, followed by step-by-step instructions for installing PyPSA-Earth.
 
+Hardware Requirements
+=====================
+Ensure that your system meets the minimum hardware specifications to run PyPSA-Earth effectively. Recommended hardware specifications may include:
+
+* 8-16 GB RAM and adequate CPU (at-least 2-cores)
+
+* Storage (HDD/SSD) capacity depends on the region of interest. Africa model requires 40 Gb, the world `--`` 250 Gb, a single country `--` between 1-10 Gb. Tutorial requires just below 10Gb. Thus, considering all required software tools, at least 40 GB of storage space is recommended.
```

Comment:
Could it be possible to run a tutorial with only a couple of Gb disc space like 2-3Gb, according to your experience? Or is 10 Gb a rather hard limit?

I recognise that such a possibility can't be of big practical meaning, as modeling would require 10 Gb, anyway. However, we still had some requests from people having very limited resources, and would be nice to give them an opportunity to have a look, at least...